gooey/gui/controller.py

```diff
@@ -108,8 +171,8 @@ def required_section_complete(self):
   def success_dialog(self):
     self.show_dialog(i18n._("execution_finished"), i18n._('success_message'), wx.ICON_INFORMATION)
 
-  def error_dialog(self, error_msg):
-    self.show_dialog(i18n._('error_title'), i18n._('uh_oh').format(error_msg), wx.ICON_ERROR)

```

Comment:
Why did you drop the error here (and in the translations file)? 

Gooey should bubble up _why_ the program failed.